src/Serilog.Sinks.Datadog.Logs/Sinks/Datadog/DatadogConfiguration.cs

```diff
@@ -40,7 +40,13 @@ public class DatadogConfiguration
         /// </summary>
         public bool UseSSL { get; private set; }
 
-        public DatadogConfiguration(string url = DDUrl, int port = DDPort, bool useSSL =  true)
+        public DatadogConfiguration()
+            : this(DDUrl, DDPort, true)
+        {
+
+        }
+
+        public DatadogConfiguration(string url, int port, bool useSSL)
```

Comment:
Why? This is a breaking change: one cannot use the 1 and 2 arguments versions of the constructor anymore. Is this a publicly usable API?

Reply:
Agreed, good catch. Yes `DatadogConfiguration` is used client side to configure the logger